Documentation/user-guides/shards-and-replicas.md

```diff
@@ -0,0 +1,154 @@
+# Shards and Replicas
+
+If single prometheus can't hold current targets metrics,user can reshard targets on multiple prometheus servers.
+Shards use prometheus `modulus` configuration to implement, which take of the hash of the source label values, split scrape targets based on the number of shards.
+
+Prometheus operator will create number of `shards` multiplied by `replicas` pods.
+
+Note that scaling down shards will not reshard data onto remaining instances,it must be manually moved. Increasing shards will not reshard data either but it will continue to be available from the same instances.
```

Comment:
```suggestion
Note that scaling down shards will not reshard data onto remaining instances, it must be manually moved. Increasing shards will not reshard data either but it will continue to be available from the same instances.
```